tests/Microsoft.Bot.Schema.Tests/Microsoft.Bot.Schema.Tests.csproj

```diff
@@ -13,6 +13,7 @@
 	</ItemGroup>
 
 	<ItemGroup>
+		<ProjectReference Include="..\..\libraries\Microsoft.Bot.Builder\Microsoft.Bot.Builder.csproj" />
```

Comment:
Q: Why do we need to reference BotBuilder in this project?

Reply:
We added it so we can use the MessageFactory class, but we can refactor the test to remove it.

Reply:
That would be good so we don't depend on that project at the Schema level.

Thanks

Reply:
Ok, we'll have this for tomorrow. Thanks!

Reply:
Changes applied. Thanks!